src/components/TableBody.js

```diff
@@ -105,10 +105,23 @@ class TableBody extends React.Component {
   };
 
   handleRowClick = (row, data, event) => {
-    // don't trigger onRowClick if the event was actually a row selection
+    // Don't trigger onRowClick if the event was actually a row selection
     if (event.target.id && event.target.id.startsWith('MUIDataTableSelectCell')) {
       return;
     }
+
+    // Check if we should trigger row expand/select when row is clicked anywhere
```

Comment:
Triggering both select and expand at the same time is UX I believe we should avoid. What I recommend is adding a second option for `selectableRows` called `selectableRowsOnClick` that will provide this, similar to `expandableRows`, so that the options are:

- `expandableRowsOnClick: true, selectableRowsOnClick: false`: Only expandable rows are opened on click
- `expandableRowsOnClick: false, selectableRowsOnClick: true`: Only selectable rows are selected on click
- `expandableRowsOnClick: false, selectableRowsOnClick: false`: Default behavior
- `expandableRowsOnClick: true, selectableRowsOnClick: true`: Both expandable rows and selectable rows are activated on click, provided that those options are enabled.